compose/compose-file/compose-file-v1.md

```diff
@@ -0,0 +1,663 @@
+
+    162.242.195.82  somehost
+    50.31.209.229   otherhost
+
+### image
+
+Specify the image to start the container from. Can either be a repository/tag or
+a partial image ID.
+
+    image: redis
+    image: ubuntu:14.04
+    image: tutum/influxdb
+    image: example-registry.com:4000/postgresql
+    image: a4bc65fd
+
+If the image does not exist, Compose attempts to pull it, unless you have also
+specified [build](#build), in which case it builds it using the specified
+options and tags it with the specified tag.
+
+> **Note**: In the [version 1 file format](#version-1), using [`build`](#build) together
+> with `image` is not allowed. Attempting to do so results in an error.
+
+### labels
+
+Add metadata to containers using [Docker labels](/engine/userguide/labels-custom-metadata.md). You can use either an array or a dictionary.
+
+It's recommended that you use reverse-DNS notation to prevent your labels from conflicting with those used by other software.
+
+    labels:
+      com.example.description: "Accounting webapp"
+      com.example.department: "Finance"
+      com.example.label-with-empty-value: ""
+
+    labels:
+      - "com.example.description=Accounting webapp"
+      - "com.example.department=Finance"
+      - "com.example.label-with-empty-value"
+
+### links
+
+Link to containers in another service. Either specify both the service name and
+a link alias (`SERVICE:ALIAS`), or just the service name.
+
+    web:
+      links:
+       - db
+       - db:database
+       - redis
+
+Containers for the linked service will be reachable at a hostname identical to
+the alias, or the service name if no alias was specified.
+
+Links also express dependency between services in the same way as
+[depends_on](#dependson), so they determine the order of service startup.
+
+> **Note:** If you define both links and [networks](#networks), services with
+> links between them must share at least one network in common in order to
+> communicate.
+
+### log_driver
+
+> [Version 1 file format](compose-versioning#version-1) only. In version 2 and up, use
+> [logging](index.md#logging).
+
+Specify a log driver. The default is `json-file`.
+
+    log_driver: syslog
+
+### log_opt
+
+> [Version 1 file format](compose-versioning#version-1) only. In version 2 and up, use
+> [logging](index.md#logging).
+
+Specify logging options as key-value pairs. An example of `syslog` options:
+
+    log_opt:
+      syslog-address: "tcp://192.168.0.42:123"
+
+### net
+
+> [Version 1 file format](#version-1) only. In version 2 and up, use
+> [network_mode](index.md#networkmode) and [networks](index.md#networks).
+
+Network mode. Use the same values as the docker client `--net` parameter.
+The `container:...` form can take a service name instead of a container name or
+id.
+
+    net: "bridge"
+    net: "host"
+    net: "none"
+    net: "container:[service name or container name/id]"
+
+#### aliases
```

Comment:
Yes, aliases. They are under the `network` key, which doesn't exist in V1, net is only a single value